lib/src/map/controller/map_controller_impl.dart

```diff
@@ -236,11 +239,14 @@ class MapControllerImpl extends ValueNotifier<_MapControllerState>
     }
 
     if (degree == camera.rotation) {
-      return const (moveSuccess: false, rotateSuccess: false);
+      return const MoveAndRotateResult(
```

Comment:
Can do, but instead of an AND I'd use an OR because as far as I understood the return is false if the current camera already matched the requested chances and therefore true if any change has made to the camera, not specifically the requested (in case of max map bounds).
Chaining both results together with OR would mean "some change has been made" and not specifically "the requested change".

Side note: I'd prefer to have only one function for all move, zoom and rotate gestures and the function like a `changeCamera` function with optional parameters for rotation, center and zoom.

Reply:
> Chaining both results together with OR would mean "some change has been made" and not specifically "the requested change".

Yep, that seems more sensible.

> Side note: I'd prefer to have only one function for all move, zoom and rotate gestures and the function like a changeCamera function with optional parameters for rotation, center and zoom.

That sounds sensible.

Reply:
Removed `MoveAndRotateResult`.

I'd prefer to have a combined `changeCamera` method not as part of this pr.